client/scripts/controllers/server/threads.ts

```diff
@@ -553,7 +553,7 @@ class ThreadsController {
       linking_thread_id,
       linked_thread_id,
       address: app.user.activeAccount.address,
-      author_chain: app.user.activeAccount.chain,
+      author_chain: app.user.activeAccount.chain.id,
```

Comment:
it doesn't seem like `address` or `author_chain` ever get used in the route? let's remove them and then get this PR in.

Reply:
Nah they get used for the checks. `lookupAddressIsOwnedByUser` requires it.

Actually, we should just standardize when we use the `lookup` helpers and what data they need. And maybe just pass it with req.user or someth.